src/selectors/selectMainWebviewViewProps.ts

```diff
@@ -55,6 +56,16 @@ export const selectMainWebviewViewProps = (
 		};
 	}
 
+	if (state.activeTabId === 'sourceControl') {
+		const sourceControlTabProps = selectSourceControlTabProps(state);
+
+		return {
+			activeTabId: state.activeTabId,
+			title: sourceControlTabProps?.title ?? null,
```

Comment:
I'd make it an empty string, or return `selectSourceControlTabProps` as the view props so we show what has happened in the app

Reply:
addressed